lib/tropic/tropic_paragraph.flow

```diff
@@ -225,9 +240,19 @@ reflowTParaWords(
 	generalIndent : double,
 	linesCountB : DynamicBehaviour<int>,
 	alignWidthM : Maybe<double>
-) -> Pair<WidthHeight, double> {
+) -> Pair<Pair<WidthHeight, double>, WordsAndIndexes> {
```

Comment:
Introduce special data type instead of nested Pair. it looks bad and it creates two objects instead of one so it is bad for GC too